src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -1356,15 +1356,15 @@ def report_timeout(self):
             # dump engines ULT stacks upon test timeout
             self.dump_engines_stacks("Test has timed-out")
 
-    def fail(self, message=None):
+    def fail(self, msg=None):
         """Dump engines ULT stacks upon test failure."""
         self.dump_engines_stacks("Test has failed")
-        super().fail(message)
+        super().fail(msg)
 
-    def error(self, message=None):
+    def error(self, msg=None):
```

Comment:
(pylint-arguments-renamed) Parameter 'msg' has been renamed to 'self' in overridden 'TestWithServers.error' method